Update app-service-mobile-node-backend-how-to-use-server-sdk.md

dhei opened the pull request:
Add a note on avoiding system properties name conflicts on Azure Mobile Apps Easy Tables doc.

add @elamalani for review.

azurecla commented:
Hi __@tmullaney__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

dhei committed:
Update app-service-mobile-node-backend-how-to-use-server-sdk.md

Add a note on avoiding system properties name conflicts on Azure Mobile Apps Easy Tables doc
